package-system/SPIRVCross/build_config.json

```diff
@@ -42,16 +27,6 @@
       "Darwin":{
          "Mac":{
             "cmake_find_template":"FindSPIRVCross.cmake.Mac",
-            "cmake_generate_args_debug":[
-               "-G",
-               "Xcode",
-               "-DCMAKE_OSX_DEPLOYMENT_TARGET=10.13",
-               "-DSPIRV_CROSS_CLI=ON",
-               "-DSPIRV_CROSS_SHARED=OFF",
-               "-DCMAKE_INSTALL_LIBDIR=\"lib/Debug\"",
-               "-DCMAKE_INSTALL_BINDIR=\"bin/Debug\"",
-               "-DCMAKE_BUILD_TYPE=Debug"
-            ],
```

Comment:
Was this needed to be removed? This is causing the script to fail, it needs at least the generator `-G` argument

Reply:
I have re-added it back as a debug config is needed.